### ISSUE TITLE
forecast vs fable comparison

### ISSUE THREAD

**Issue by emitanaka** [Yiwen-Zhang-259/ETC5543-cran-analysis#7]
@Yiwen-Zhang-259 when you get a chance could you have a look also at the download logs for `fable` and `forecast`. 
I know (at least for a subset of CRAN packages), older packages generally have more download (like say in the last two years) than newer packages (you can plot "total downloads over last 2 years" vs. "Date when package got released" to explore this). I suspect though that the rate of recent download is quite different for these with `fable` getting far more download in recent times. Note that `pkgsearch:: cran_trending` can find trending packages. This information might be more interesting than looking at total downloads and be an alternative indicator of how impactful a package is.

**Comment by Yiwen-Zhang-259** [comment]
Ok Emi, that makes sense, thank you !

**Comment by Yiwen-Zhang-259** [comment]
Oh, after ploting, I found that `fable` is older than `forecast` (by checking the published date from CRAN website), but the download count of `forecast` is much more than that of `fable` in last two years.

**Comment by Yiwen-Zhang-259** [comment]
> @Yiwen-Zhang-259 when you get a chance could you have a look also at the download logs for `fable` and `forecast`.
> I know (at least for a subset of CRAN packages), older packages generally have more download (like say in the last two years) than newer packages (you can plot "total downloads over last 2 years" vs. "Date when package got released" to explore this). I suspect though that the rate of recent download is quite different for these with `fable` getting far more download in recent times. Note that `pkgsearch:: cran_trending` can find trending packages. This information might be more interesting than looking at total downloads and be an alternative indicator of how impactful a package is.

Oh, after ploting, I found that `fable` is older than `forecast` (by checking the published date from CRAN website), but the download count of `forecast` is much more than that of `fable` in last two years.

**Comment by emitanaka** [comment]
Yup I know. `fable` is the tidy version of `forecast` and got on CRAN only recently. You shouldn't compare the raw download counts directly since it's natural that `forecast` will have much higher download. What about the rate of download? Look into `pkgsearch::cran_trending` if you need some inspiration to think of different ways of comparing.

**Comment by Yiwen-Zhang-259** [comment]
> Yup I know. `fable` is the tidy version of `forecast` and got on CRAN only recently. You shouldn't compare the raw download counts directly since it's natural that `forecast` will have much higher download. What about the rate of download? Look into `pkgsearch::cran_trending` if you need some inspiration to think of different ways of comparing.

ok thank you Emi
